crates/nu-path/src/lib.rs

```diff
@@ -6,12 +6,14 @@ pub mod expansions;
 pub mod form;
 mod helpers;
 mod path;
+pub mod pwd_per_drive;
```

Comment:
Seems like you could do `#[cfg(windows)]` before `pub mod pwd_per_drive` to restrict it to windows only.

Reply:
OK, continue improving, :)